tests/models/decoder_only/vision_language/vlm_utils/types.py

```diff
@@ -52,6 +52,8 @@ class SizeType(Enum):
 class CustomTestOptions(NamedTuple):
     inputs: List[Tuple[List[str], List[Union[List[Image], Image]]]]
     limit_mm_per_prompt: Dict[str, int]
+    # kwarg to pass multimodal data in as to vllm/hf runner instances.
+    runner_mm_key: str = "images"
```

Comment:
Is this information actually necessary? I think it can be inferred from `VLMTestType`.

Reply:
I agree, that's actually the intent of moving it here. Since this was in `VLMTestInfo`, I realized that it's awkward to keep it at that level if a model supports multiple modalities. It is moved here so that `images` is always used for `VLMTestTypes` Image/MultiImage/Embedding, and `videos` is always used for `Videos`, and it can only be set for CustomInputs to avoid mistakes while writing tests.

An example of when this would be needed to make things feel less clunky would be if we had a couple long context tests for qwen2-vl for images/video respectively to test the ongoing issues, because we could just write something like this:

```python
        custom_test_opts=[
            CustomTestOptions(
                 inputs=custom_inputs.get_long_context_qwen_vl_video_inputs(),
                 limit_mm_per_prompt={"video": 4},
                 runner_mm_key="videos",
             ),
            CustomTestOptions(
                 inputs=custom_inputs.get_long_context_qwen_vl_image_inputs(),
                 limit_mm_per_prompt={"image": 4},
                 runner_mm_key="images",
             ),
    ],
```

to configure those edge-cases separately